pkg/templates/openapi/ext.go

```diff
@@ -20,6 +20,10 @@ const (
 	ExtOriginalType              = "type"
 	ExtOriginalValueExpression   = "value-expression"
 	ExtOriginalVariablesSequence = "sequence"
+
+	// Extension for constraint.
+	ExtConstraint        = "x-walrus-constraint"
```

Comment:
Previously, I thought it would be easier to do this, but in the future, there may be other constaints, such as the operating system

Reply:
If we need such extensibility, can we use `x-walrus` instead of `x-walrus-constaint` so that it's more extensible? Similar to 
```
terraform {
   version = ">= 1.2.0, < 2.0.0"
}
```

I don't understand how os contraint works at this moment.